primitives/transaction-pool/src/error.rs

```diff
@@ -25,49 +25,48 @@ use sp_runtime::transaction_validity::{
 pub type Result<T> = std::result::Result<T, Error>;
 
 /// Transaction pool error type.
-#[derive(Debug, derive_more::Display, derive_more::From)]
+#[derive(Debug, thiserror::Error, derive_more::From)]
```

Comment:
what is `derive_more` used for?

Reply:
`thiserror::Error` with `#[from]` only is allowed to be used for inner types, that impl `std::error::Error + Send + ..` bounds. But here this is not the case for any. So here the `derive_more::From` came in handy :+1:

Reply:
makes sense @drahnr